rails/app/controllers/api/v1/report_learners_es_controller.rb

```diff
@@ -44,16 +45,22 @@ def external_report_query
     # In the future, we might want to extend this query format and add other filters, e.g. dates.
     query = {
       type: "learners",
-      version: "1.0",
+      version: "1.1",
       learners: select_learners.map do |l|
         {
           run_remote_endpoint: l.remote_endpoint_url,
-          class_id: l.offering && l.offering.clazz_id
+          class_id: l.offering && l.offering.clazz_id,
+          runnable_url: l.offering && l.offering.runnable_type == "ExternalActivity" ? l.offering.runnable.url : nil
         }
-      end
+      end,
+      user: {
+        id: url_for(current_user),
+        email: current_user.email
+      },
+      reportServiceSource: ENV['REPORT_SERVICE_SOURCE']
```

Comment:
Is it necessary to have this be part of the signed query?
If this just needs to be configured it could be part of the URL that is setup for the external report.